tsl/src/continuous_aggs/create.c

```diff
@@ -1155,6 +1155,15 @@ cagg_validate_query(const Query *query, const bool finalized, const char *cagg_s
 		{
 			cagg_source = ts_continuous_agg_find_by_relid(rte->relid);
 
+			if (!cagg_source)
```

Comment:
Can u rename this var to "cagg_parent"? Is a better name and I forgot to rename it in the original PR. I'm using "parent" in several places already!